packages/calcite-components/src/components/flow-item/flow-item.scss

```diff
@@ -3,22 +3,36 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-flow-item-background-color: Specifies the background of the component.
+ * @prop --calcite-flow-item-color: Specifies the color of the component.
```

Comment:
Do we want `flow-item-header-color` and `flow-item-description-color` here?

Reply:
not unless there have been requests for unique styling for those elements.